app/src/main/java/fr/free/nrw/commons/Achievements.java

```diff
@@ -0,0 +1,104 @@
+package fr.free.nrw.commons;
```

Comment:
I think it would be a good idea to create an Achievements package for your GSoC project, as you will likely create some helper classes along the way. Useful to have them all in 1 package.

Reply:
Yes I will modify it in my next update 👍

Reply:
Great! :) One more thing - as this is an activity, we should follow conventional naming schemes, i.e. "AchievementsActivity.java".